_pages/en_US/include/3ds-11.17.txt

```diff
@@ -2,5 +2,4 @@
 
 + If your console is running *Luma 10.2.1 or higher*, it is *100% safe* to update. You can check your Luma version by holding (Select) while booting your console.
 + If your console is on an older Luma version, you should [update Luma](checking-for-cfw) before you update your console to 11.17.0.
-+ If your console does not yet have custom firmware, **do NOT update to the latest version** unless this guide instructs you to. The methods on earlier firmwares are easier and more stable.
-+ If your unmodified console is ALREADY on 11.17.0, this guide has been updated accordingly. Both [New3DS](installing-boot9strap-(super-skaterhax)) and [Old3DS](installing-boot9strap-(mset9)) now have free software methods that (for the most part) do not require external hardware or software.
++ If your console does not yet have custom firmware, you can install it now on all firmwares with this guide.
```

Comment:
I think it's been long enough that this include could be removed entirely

Reply:
dont disagree, but maybe it doesnt belong into this pr

Reply:
Would be cool. Just added the change because super-skaterhax is gone with this

Reply:
I think the include should be kept but removed from keyinfo maybe?